login-workflow/docs/language-support.md

```diff
@@ -8,6 +8,101 @@ This package supports translations to different languages using [i18next](https:
 -   Spanish
 -   Simplified Chinese
 
-## Changing the Language
+## Add/Override translations within our workflow
 
-<!-- @TODO: Finish this... -->
\ No newline at end of file
+To add i18n in your application you can follow the instructions from [official website](https://react.i18next.com/getting-started).
+You need to wrap your app with <I18nextProvider/> for rendering translations in the app. For reference, you can check [I18nextProvider](https://react.i18next.com/latest/i18nextprovider).
+
+
+If you want to use keys from your application dictionary in any of the workflow screens or any custom screens that you provide within the Auth or Login workflows, you need to pass your app's i18n instance to the `AuthContextProvider` via the `i18n` prop.
+
+```tsx
+import { i18nAppInstance } from './i18n';
+
+<AuthContextProvider
+    ...
+    i18n={i18nAppInstance}
+>
+```
+
+`Registration workflow` works in the same way, allowing you to pass an i18n instance via the `i18n` prop.
+
+```tsx
+import { i18nAppInstance } from './i18n';
+
+<RegistrationContextProvider
+    ...
+    i18n={i18nAppInstance}
+>
+```
+f you need to override any of the translations used internally in any of the components or screens, you can use the `t` function from `react-i18next` and pass the translation key.
+For e.g. if you want to override default `Email` label of `<ForgotPasswordScreen />` component you can pass key `EMAIL` to `t` function without namespace. It will take default namespace which you declare while creating an i18n instance.
+```tsx
+import { useTranslation } from 'react-i18next';
+
+const { t } = useTranslation();
+<ForgotPasswordScreen emailLabel={`${t('EMAIL')}`} />
+```
+
+## Add New language within our workflow
+
+To add new languages other than English, French, Portuguese, Spanish, and Simplified Chinese. You need to add your own translation dictionary which will have our existing translation keys.
+
+To add new language with all translation keys of workflow you can do it same as below. The below example is only having a few keys to our workflow. You need to use all resource IDs to translate the complete workflow.
```

Comment:
This might not be 100% correct but maybe some more word smithing is needed.

To add a new language using the translation keys that exists, simply create new dictionary files in AuthDictionaries and SharedDictionaries for the language needed. The example below shows how to start implementing the usage of the new language in your App and requires all resource IDs to translate the entire application workflow. Refer to the language files located in the dictionaries to build out the entire language translations.

Reply:
ok wasn't sure. So steps 1 through ...5 what is all needed for end user to add German to the entire workflow?

Reply:
They need to create their own new dictionary in the language of their choice that matches the structure of our internal dictionary (link to the structure so they can copy it). The provide the translations for every key that the workflow screens need and pass that dictionary in to their i18n instance, which in turn gets passed through the i18n prop to our workflow context providers.

Reply:
Mentioned here https://github.com/etn-ccis/blui-react-workflows/pull/384/files#diff-06fa8d60640f80c68f6f43eb7e01a693293e0c02d66578196aeb1ca9cc6b317eR53